src/Bicep.Core/Analyzers/Linter/Rules/ParametersRequiredRule.cs

```diff
@@ -11,12 +11,12 @@ namespace Bicep.Core.Analyzers.Linter.Rules
 {
     public sealed class ParametersRequiredRule : LinterRuleBase
     {
-        public new const string Code = "Parameters Required";
+        public new const string Code = "parameters-required";
         public ParametersRequiredRule() : base(
             code: Code,
             description: CoreResources.ParameterRequiredRuleDescription,
             diagnosticLevel: Diagnostics.DiagnosticLevel.Warning,
-            docUri: "https://bicep/linter/rules/BCPL1000") //TODO: set up online documentation location
+            docUri: "https://aka.ms/linter-rules")
```

Comment:
What I meant was to have individual aka.ms's, e.g. aka.ms/bicep/linter/params-required etc.  But if you wanted you could make them all temporarily point to the same doc.  But since you have different docs anyway that doesn't make sense, sorry.

I'd sugges aka.ms/bicep/linter/params-required etc.

Reply:
or aka.ms/bicep/linter/rules/params-required